### ISSUE TITLE
SmallRye: Compatible Certification Request for MicroProfile Fault Tolerance 3.0 

### ISSUE THREAD

**Issue by Ladicek** [eclipse/microprofile-fault-tolerance#586]
- [x] Organization Name ("Organization") and, if applicable, URL:<br/>
  Red Hat

- [x] Product Name, Version and download URL (if applicable):<br/>
  [SmallRye Fault Tolerance 5.0.0-RC1](https://github.com/smallrye/smallrye-fault-tolerance/releases/tag/5.0.0-RC1)

- [x] Specification Name, Version and download URL:<br/>
   [MicroProfile Fault Tolerance 3.0](https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/3.0)

- [ ] (Optional) TCK Version, digital SHA-256 fingerprint and download URL:<br/>

- [x] Public URL of TCK Results Summary:<br/>
  https://github.com/smallrye/smallrye-fault-tolerance/actions/runs/413322319

- [ ] Any Additional Specification Certification Requirements:<br/>

- [x] Java runtime used to run the implementation:<br/>
  Java 8 and 11

- [x] Summary of the information for the certification environment, operating system, cloud, ...:<br/>
  GH Actions - Ubuntu 

- [x] By checking this box I acknowledge that the Organization I represent accepts the terms of the [EFTL](https://www.eclipse.org/legal/tck.php).

- [x] By checking this box I attest that all TCK requirements have been met, including any compatibility rules.

**Comment by Ladicek** [comment]
(I had to create the label manually, it wasn't there yet :-) )

**Comment by kwsutter** [comment]
The Specification Name and Version item should point at the actual Spec. I would put a reference to the eventual maven repo: https://search.maven.org/artifact/org.eclipse.microprofile.fault-tolerance/microprofile-fault-tolerance-spec/3.0/pom

Same thing for the Compatible Impl. Reference the maven artifact: https://search.maven.org/artifact/io.smallrye/smallrye-fault-tolerance/5.0.0-RC1/jar or have the binary available in a download directory that will not be removed. We need a "permanent" location for the CI.

**Comment by radcortez** [comment]
Updated.

**Comment by Emily-Jiang** [comment]
> The Specification Name and Version item should point at the actual Spec. I would put a reference to the eventual maven repo: https://search.maven.org/artifact/org.eclipse.microprofile.fault-tolerance/microprofile-fault-tolerance-spec/3.0/pom

@kwsutter This depends on the purpose of this CCR. If we put the unexisting repo as your specified, people won't be able to click the link to verify. This might have defeated the purpose of this CCR. CCR is not the certification proof from my understanding. It is the evidence to prove we can pass all TCKs so we can release the artificates. With what your suggestions, it is like you want this issue hangs around after the release. I think all links should be clickable. I think the link to the relevant repo release tag is the permanent link and it always works even after the push to maven central happens (e.g. https://github.com/eclipse/microprofile-config/releases/tag/2.0)

**Comment by Emily-Jiang** [comment]
Once we agree on this, we need to put the requirement on the templates for future releases.

**Comment by kwsutter** [comment]
@Emily-Jiang, all CCRs carry the same weight.  So, whether we are using it for the first Compatible Implementation that goes with the Spec release, or if we're using it for the 10th Compatible Implementation, it shouldn't matter.  I wouldn't treat this CCR like it's "throw away".  We should be putting data in here that can be referenced long down the road.  Actually, this first CCR is probably even more important since this is the one that we are using to certify the Specification release.

You are correct that the final URL to the Spec will not work until this is release is completed.  But, that's okay.  We're reviewing all of the material as part of this ballot.  So, as a reviewer, I can verify that the URL is correct even if it doesn't work until the release is done.

BTW, this approach is consistent with what we have been doing with Jakarta EE.  And, it seems to be working...

**Comment by dblevins** [comment]
Similar feedback as @kwsutter.  See my same comment on this CCR https://github.com/eclipse/microprofile-open-api/issues/459#issuecomment-748362004

**Comment by Ladicek** [comment]
If we need to be precise, then technically search.maven.org is not where the artifacts are, and we should point to repo.maven.apache.org :-)

**Comment by Emily-Jiang** [comment]
@Ladicek good point! I think it is better to point to repo.maven.apache.org. Thoughts? @kwsutter
